VD-10

ahackit opened the pull request:
This PR adds the initial readme for some basic information to get going with links back to the full documentation.

The full documentation is a minimal nuxt/content setup using the nuxt-content-docs theme. Documentation is provided via markdown files in the content directory. 

To test and view readme - https://github.com/loftylabs/vue_deckgl/tree/vd-10
To test and view documentation
- checkout branch
- cd docs
- npm install.
- npm run dev
- access docs at localhost:3000

The only thing left to do is add hosting for the docs, a couple of options are being explored currently by @TyrelDenison. Once added, will set it up into CI/CD pipeline

ahackit committed:
Update readme.

ahackit committed:
Updates Readme + Adds Nuxt Content and updates gitignore.

wsmoyer reviewed (commented):
on those installation instructions, you might as well add the deck.gl package install itself since they will have to install it to import layers

ahackit commented:
> on those installation instructions, you might as well add the deck.gl package install itself since they will have to install it to import layers

@wsmoyer Will test again, but I'm pretty sure since it's a dependency you have access to it already.

TyrelDenison reviewed (approved):
Nuxt looks good

🍰

ahackit committed:
Moves docs to _docs so we can build to docs folder for github pages.
Updates nuxt settings to accomodate new base url.
Adds default docs folder.
Updates gitignore.
Updates circleci for docs deploys.
